### DISCUSSION TITLE
Multiple clones per cell barcode

### DISCUSSION THREAD

**Discussion by SirKuikka** [milaboratory/mixcr#1796] [General]
Hi, 

I have a simple question. I am looking at the .clones.tsv output for the single-cell Mixcr workflow, and there are often multiple clones per cell barcode. Should I just use the first one with the most reads/UMIs if I want to assign a clone identity to each cell barcode?

![image](https://github.com/user-attachments/assets/1bb4ae45-5038-49e7-b9bd-2fd6356b642c)

![image](https://github.com/user-attachments/assets/95319aaf-26fc-4997-8aa6-c82e61bf0316)

**Comment by mizraelson** [comment]
Hi, what MiXCR version and command do you use?

**Reply by SirKuikka** [reply]
Hi, 

v4.5.0

mixcr analyze %%s -s hsa --rna --threads 4 --force-overwrite %%s_S1_L{{n}}_{{R}}_001.fastq.gz %%s/%%s'%%(preset,gsm,outdir,gsm)

preset was 10x-sc-5gex

The sample I am analyzing is this. https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM5264643

**Reply by mizraelson** [reply]
Could you try running the latest version. There should be no more than 2 clones per chain per cell.
